eos-split-flatpak-repo

```diff
@@ -379,7 +494,7 @@ def split_repo(prune=False, root='/'):
     if not os.path.islink(flatpak_dir_path):
         logger.info('Flatpak dir %s is not a symlink', flatpak_dir_path)
     else:
-        changed = True
+        stats.changed = True
```

Comment:
Hmm. So in the very unlikely case that the block above succeeds and then the script is interrupted or fails in this block, we would end up not submitting the stats on that run, and on a subsequent run we would submit a stub event with most fields being `0`. I think that's OK though.

Reply:
Yes, this was really bugging me but I don't know exactly what to do about it. I guess you could treat this part as a separate result or just leave it out of the results completely.

Reply:
I think "do nothing" because the chances of hitting this path are so tiny

Reply:
(and if I'm wrong about that we'll know from the data – it will be easy to distinguish)